website/docs/components/icon/partials/code/how-to-use.md

```diff
@@ -0,0 +1,109 @@
+## How to use this component
+
+The most basic invocation of the Icon component requires only the `@name` property to be passed with a value matching an existing name in [the Icon library](/icons/library):
+
+```handlebars
+<Hds::Icon @name="alert-circle" />
+```
+
+It renders to this (where the `id` will be unique each time):
+
+```markup
+<svg
+    id="icon-ember115"
+    class="hds-icon icon-alert-circle"
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
+    xmlns="http://www.w3.org/2000/svg"
+    fill="currentColor"
+    aria-hidden="true"
+    data-test-icon="alert-circle"
+>
+    <use href="/@hashicorp/flight-icons/icons/sprite.svg#alert-circle-16"></use>
+</svg>
+```
+
+Because the icons are hidden to assistive technology, they cannot be used on their own and must be used inside of an element with an accessible name. See the [Accessibility](/components/icon?tab=accessibility) section for more details on how to best use this component in different contexts.
+
+### Size
+
+The default size is 16px. To use the alternative 24px icon size, set the `@size` value:
+
+```handlebars
+<Hds::Icon @name="zap" @size="24" />
+```
+
+### Color
+
+The default value is `currentColor` which uses the inherited text color as the icon color. When setting a custom value, we recommend using one of the pre-defined variables to ensure consistency with our design language:
+
+```handlebars
+<Hds::Icon @name="zap" @color="var(--token-color-foreground-success)" />
+```
+
+Other accepted values include named colors and color values themselves (e.g., hex, rgb, etc).
+
+```handlebars
+<Hds::Icon @name="zap" @color="rebeccapurple" />
+```
+
+```handlebars
+<Hds::Icon @name="zap" @color="rgb(46, 113, 229)" />
+```
+
+### Stretched
+
+To have the icon fill the parent container (width: 100%, height: 100%), set the `@stretched` attribute to true:
+
+```handlebars
+<div class="doc-icon-demo--constrain-max-width">
+    <Hds::Icon @name="zap" @size="24" @stretched={{true}} />
+</div>
+```
+
+### isInline
```

Comment:
thought: with very few exceptions (that would need to be rectified), we tend to use 'human-friendly' titles for our sections; what about 'Inline' or 'Block vs. Inline' or 'Block or Inline'

Reply:
or we can even move this example within the 'Aligning icons' section; in the end it's about aligning and we refer to it in the section anyway

Reply:
I would change this title to "Inline vs. block display" or something along those lines.

Reply:
```suggestion
### Block vs. inline display
```